scene/animation/animation_blend_tree.cpp

```diff
@@ -932,8 +942,12 @@ void AnimationNodeTransition::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_xfade_curve", "curve"), &AnimationNodeTransition::set_xfade_curve);
 	ClassDB::bind_method(D_METHOD("get_xfade_curve"), &AnimationNodeTransition::get_xfade_curve);
 
+	ClassDB::bind_method(D_METHOD("set_transition_to_self", "enable"), &AnimationNodeTransition::set_transition_to_self);
+	ClassDB::bind_method(D_METHOD("is_transition_to_self"), &AnimationNodeTransition::is_transition_to_self);
```

Comment:
Should be either `is_transitioning_to_self` or `get_transition_to_self` (maybe the latter is better, the former sounds like the process is ongoing).

Reply:
Or `is_transition_to_self_enabled`... We never really reached consensus on https://github.com/godotengine/godot-proposals/issues/1994